### PR TITLE
feat: [CI-14261]: Added support of hardware acceleration in GCP VMs

### PR DESCRIPTION
**Pull request by ShobhitSingh11**
# Commit Checklist

Thank you for creating a pull request! To help us review / merge this can you make sure that your PR adheres as much as possible to the following.

## The Basics

If you are adding new functionality, please provide evidence of the new functionality.

### CODE REVIEW

File: store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE instances ADD COLUMN IF NOT EXISTS enable_nested_virtualization BOOLEAN NOT NULL DEFAULT false;
```
Comment (raghavharness) [inline store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql:1]: do we need to store in database ? why ?

Comment (ShobhitSingh11) [reply store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql:1]: Reason is we need to log during init whether nedtedVirt is enabled or not, for that we are storing nestedVirt variable in instance property and use this instance property to show it in log. For pools > 0 without storing it in db that instance variable is not populating properly (was always showing false, even if nestedVirt is actually enabled in instance). However instance creation was working properly according to the property set in pool.yml

**Review by raghavharness — COMMENTED** [review]
Looks good, please add in the grafana metrics as well

File: store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql (hunk 1 above)
Comment (raghavharness) [reply store/database/migrate/postgres/0011_update_table_instances_nested_virtualization.up.sql:1]: add enable_nested_virtualization property in grafana metrics also

**Review by raghavharness — APPROVED** [review]

**Pull request merged**